src/doc/reference.md

```diff
@@ -2427,13 +2427,13 @@ There are three different types of inline attributes:
 
 ### Deriving
```

Comment:
I've tended to just make it

    ## `derive`

or similar, when a heading is talking about a specific feature. The section of `if` does this, for example